terroroftinytown/tracker/model.py

```diff
@@ -436,22 +436,30 @@ def has_results(cls):
             return bool(result)
 
     @classmethod
-    def get_count(cls):
+    def get_count(cls, project_id=None, **unused_kwargs):
         with new_session() as session:
-            min_id = session.query(func.min(Result.id)).scalar() or 0
-            max_id = session.query(func.max(Result.id)).scalar() or 0
+            q = session.query(func.count(Result.id))
+            if project_id is not None and project_id != 'None':
+                q = q.filter(Result.project_id == project_id)
+            return q.scalar()
+
 
-            return max_id - min_id
```

Comment:
I'm not sure about `unused_kwargs`. Why does it need unused args?

The reason I didn't use the obvious `count()` was because SQLite does a full table scan. Since the result id is a integer primary key, it is also  `rowid`. So doing `max(id)` or `min(rowid)` on SQLite is O(1). Perhaps it should only use `count()` when `project_id` is provided since a full scan is required anyway.